app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductOverviewFragment.java

```diff
@@ -386,6 +394,20 @@ void toastEmbCodeHint() {
                 .show();
     }
 
+    @OnClick(R.id.hint_link)
+    void searchProductLink() {
+        String url = "https://www.google.com/search?q=" + code;
+        if (!brand.getText().toString().isEmpty()) {
+            url = url + " " + brand.getText().toString();
+        }
+        if (!name.getText().toString().isEmpty()) {
+            url = url + " " + name.getText().toString();
+        }
+        url = url + " " + "official website";
```

Comment:
this need to be translatable

Reply:
Okay! I will change this it to a string too when I will be adding all the hardcoded strings to `strings.xml`.